test/test-creation.js

```diff
@@ -296,4 +296,64 @@ describe('license:app - generate GPL-3.0 license', function () {
   });
 });
 
+describe('license:app - generate license with output option, change directory', function () {
+  before(function () {
+    return helpers.run(path.join(__dirname, '../app'))
+      .inTmpDir(function (dir) {
+        var fs = require('fs');
+        fs.writeFileSync(path.join(dir, 'package.json'), '{}');
+      })
+      .withOptions({
+        year: '2015',
+        force: true,
+        output: 'src/LICENSE'
+      })
+      .withPrompts({
+        name: 'Rick',
+        email: 'foo@example.com',
+        website: 'http://example.com',
+        license: 'GPL-3.0'
+      })
+      .toPromise();
+  });
+
+  it('creates LICENSE file using GPL-3.0 template', function () {
+    assert.fileContent('src/LICENSE', 'GNU GENERAL PUBLIC LICENSE');
```

Comment:
We care for two things here:

1. That `src/LICENSE` exists
2. That `LICENSE` doesn't

We don't care about `package.json` or the license file content in this unit case.